Demo/SPGooglePlacesAutocomplete/SPGooglePlacesAutocompleteDemoViewController.m

```diff
@@ -26,7 +29,34 @@ - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
 }
 
 - (void)viewDidLoad {
+    [super viewDidLoad];
     self.searchDisplayController.searchBar.placeholder = @"Search or Address";
+
+
+
+    // Request use on iOS 8
+    if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_7_1) {
```

Comment:
Agreed ! I was going to cleanup this since i just C/P this to make it work and be able to test quickly ! 

The mapView in iOS8 doesn't automatically ask to use location and fail silently in ios8. This allow to ask for permission and start tracking :wink: